### PR TITLE
Fix: Configdump deployment on OpenShift

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
 - Fixed incorrect label configuration. Replaced `CM_LABELS` with `CONFIG_BUMP_LABELS`
 -  ~Added `POD_NAME` that is needed for `main.go:leader.Become(context.Background(), controllerName) ` initialization~
 -  ~Added pod permissions (we can minimize this list later) for correct `main.go:manager.New`  initialization`~
 - Disabled metrics to avoid port conflicts `manager.Options{MetricsBindAddress: "0", Namespace: namespace}`
 - Added labels to configmap example

### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/17567

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,18 +89,21 @@ spec:
   containers:
   - name: traefik
     image: traefik
```
Comment (sparkoo) [inline README.md:91]:
```suggestion
    image: docker.io/traefik
```

File: README.md [hunk 2]
```diff
@@ -56,13 +56,12 @@ metadata:
   name: read-configmaps
```
Comment (sparkoo) [inline README.md:56]: this role no longer just `read-configmaps`. Maybe rename it?

File: README.md [hunk 3]
```diff
@@ -56,13 +56,12 @@ metadata:
...
 - verbs:
-  - watch
-  - get
-  - list
+  - '*'
   apiGroups:
   - ""
   resources:
   - configmaps
+  - pods
```
Comment (sparkoo) [inline README.md:64]: I'm wondering why we need these extra permissions? It was not needed for first Rust implementation.

File: README.md [hunk 4]
```diff
@@ -90,18 +89,21 @@ spec:
   containers:
   - name: traefik
     image: traefik
+    args: ["--configFile=/dynamic-config/traefik.yml"]
```
Comment (sparkoo) [inline README.md:92]: How is now `ConfigMap` with traefik.yml mounted into the pod when you removed the `config` VolumeMount?

File: README.md (hunk 3 above)
Comment (skabashnyuk) [reply README.md]:
```  go

	err = leader.Become(context.Background(), controllerName)
	if err != nil {
		return err
	}
```

this code uses it. I'll comment it.

File: README.md (hunk 2 above)
Comment (skabashnyuk) [reply README.md]: I was able to use existed roles when I removed `leader election` see https://github.com/che-incubator/configbump/pull/8#discussion_r466983219

File: cmd/configbump/main.go [hunk 5]
```diff
@@ -77,11 +75,11 @@ func initializeConfigMapController(labels string, baseDir string, namespace stri
 			}
 		}
 	}
-
-	err = leader.Become(context.Background(), controllerName)
-	if err != nil {
-		return err
-	}
+	// require extra roles with pods
```
Comment (sparkoo) [inline cmd/configbump/main.go:78]: I'd rather remove it than keep it here commented.

**Pull request merged**